serverlessworkflow/modules/ROOT/pages/core/understanding-jq-expressions.adoc

```diff
@@ -174,6 +177,21 @@ The original `numbers` array should not be returned as a result of the workflow
 
 In the previous example, the workflow model is changed by the `input` property of the filter, which means that the output model is updated before the state is executed. As a final result, the output model consists of a `result` property, containing the maximum value of `x`.
 
+=== Event data filter usage
+
+You can find an example of event data filtering in the link:{kogito_sw_examples_url}/serverless-workflow-callback-quarkus[callback workflow application]. 
+
+[source,json]
+----
+  "eventRef": "waitEvent",
+  "eventDataFilter": {
+     "data": ".result",
+     "toStateData": ".move"
+   }
+----
+
+This event filter copies the content of Cloud Event data `result` field into Workflow model `move` field. 
```

Comment:
```suggestion
The previous example of the event filter copies the content of Cloud Event data `result` field into the workflow model `move` field. 
```

Reply:
why "the previous example of the" is better than "this"?